src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -615,7 +616,7 @@ public static bool IsValidParams(Symbol member)
 
             // Note: we need to confirm the "arrayness" on the original definition because
             // it's possible that the type becomes an array as a result of substitution.
-            ParameterSymbol final = member.GetParameters().Last();
+            var final = member.GetParameters().Last();
```

Comment:
>var [](start = 12, length = 3)

Please revert this style change, we do not have a requirement to use var. In fact, we have a requirement to spell out type if type is not mentioned explicitly in initializer.